dotCMS/src/integration-test/java/com/dotcms/MainSuite1a.java

```diff
@@ -88,7 +92,11 @@
         com.dotcms.content.elasticsearch.business.ESSiteSearchAPITest.class,
         com.dotcms.content.elasticsearch.business.ContentletIndexAPIImplTest.class,
         com.dotcms.content.elasticsearch.business.ES6UpgradeTest.class,
-        com.dotcms.content.elasticsearch.business.ESContentFactoryImplTest.class
+        com.dotcms.content.elasticsearch.business.ESContentFactoryImplTest.class,
```

Comment:
why did you add the existing tests on this class? were they missing in the others MainSuiteX.java files?

Reply:
I delete the existing ones and only leave the new one.